### DISCUSSION TITLE
Migration from Kubevirt to VMware

### DISCUSSION THREAD

**Discussion by jsakil14** [kubev2v/forklift#938] [Ideas]
Is there a future approach to migrated workloads to vmware stack or do we any other tool which we can utilise if we have to go back to vmware stack after completeley moving over to kubevirt?

**Comment by ahadas** [comment]
@jsakil14 we would be willing to accept contributions for migrating to vSphere if we get such but I'm afraid we don't plan to implement that since we don't have capacity for this and VMware used to provide their own tools to migrate from KVM to vSphere (it could be that this has changed lately though, I'm not following the state of tools that are provided by VMware)
